system-addon/content-src/components/AutoMigratePrompt/AutoMigratePrompt.jsx

```diff
@@ -0,0 +1,32 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {FormattedMessage, injectIntl} = require("react-intl");
+const {actionTypes: at, actionCreators: ac} = require("common/Actions.jsm");
+
+const AutoMigratePrompt = props => {
+  if (!props.AutoMigrate.display) {
+    return null;
+  }
+
+  return props.AutoMigrate.stage === 0 ?
+    (<div className="migrate-prompt">
+      <span>{props.AutoMigrate.msg}</span>
```

Comment:
Just making sure, this `msg` should be the success message, e.g., "Dive right into Firefox! Import your favorite sites, bookmarks, history and passwords from Chrome" ? Maybe a more descriptive name will help -- `migratedMessage` ?